Makefile

```diff
@@ -2,7 +2,7 @@ phpstan:
 	APP_ENV=test bin/phpstan.sh
 
 ecs:
-	APP_ENV=test bin/ecs.sh --clear-cache
+	APP_ENV=test bin/ecs.sh --clear-cache --fix
```

Comment:
We do not want to change code (by fixing it) when running bare easy-coding-standards check.

Please revert this.